### PR TITLE
scaffold overview

### PR DESCRIPTION
**Pull request by priley86**
Preliminary changes to scaffold Overview UI for new Migrations and Infrastructure Mappings sections. Use existing Overview design styles. Creates UI elements w/ mock items for now.

* would like to revisit much of the design tweaks next week, this is just to get a scaffold in place for integrating API mocks...

migrations
.

cc: @michaelkro @vconzola @serenamarie125

### CODE REVIEW

File: app/javascript/common/API.js [hunk 1]
```diff
@@ -35,4 +35,4 @@ export default {
   }
 };
 
-export const globalMockMode = false;
+export const globalMockMode = true;
```
Comment (michaelkro) [inline app/javascript/common/API.js]: Oops?

File: app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/__tests__/__snapshots__/MappingWizardClusterStepActions.test.js.snap [hunk 2]
```diff
@@ -49,6 +49,115 @@ Array [
...
+            "updated_on": "2017-11-21T23:03:49Z",
+            "zone_id": "10000000000008",
+          },
+        ],
+        "subcount": 8,
+        "subquery_count": 8,
+      },
+    },
+    "type": "FETCH_V2V_SOURCE_CLUSTERS_FULFILLED",
+  },
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/__tests__/__snapshots__/MappingWizardClusterStepActions.test.js.snap]: This snap update is due to `mockMode` being true

File: app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/__tests__/__snapshots__/MappingWizardClusterStepActions.test.js.snap [hunk 3]
```diff
@@ -101,5 +210,93 @@ Array [
...
+            "updated_on": "2017-02-26T03:18:27Z",
+            "zone_id": "10000000000003",
+          },
+        ],
+        "subcount": 1,
+        "subquery_count": 1,
+      },
+    },
+    "type": "FETCH_V2V_TARGET_CLUSTERS_FULFILLED",
+  },
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/__tests__/__snapshots__/MappingWizardClusterStepActions.test.js.snap]: ditto

File: app/javascript/react/screens/App/Overview/__test__/__snapshots__/OverviewActions.test.js.snap [hunk 4]
```diff
@@ -0,0 +1,3201 @@
...
+                        "started_on": "2018-02-26 11:14:11 UTC",
+                        "status": "finished",
+                        "updated_on": "2018-02-26 11:14:13 UTC",
+                        "weight": 1,
+                      },
+                      "/State4": Object {
+                        "description": "Acquire Transformation Host",
+                        "message": "State4 is finished.",
+                        "percent": 100,
+
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Overview/__test__/__snapshots__/OverviewActions.test.js.snap]: Another `mockMode` snap update

File: app/javascript/common/API.js (hunk 1 above)
Comment (priley86) [reply app/javascript/common/API.js]: let's leave it true until Monday/Tuesday next week when APIs are ready... ;)

**Comment by priley86** [comment]
rebased #176...reupdated snaps...

**Pull request merged**